### DISCUSSION TITLE
Ambient lighting and reflections

### DISCUSSION THREAD

**Discussion by ericwood73** [google/model-viewer#2595] [Q&A] [answered]
According to https://developers.google.com/ar/develop/java/light-estimation, ARCore supports lighting and reflections sampled from the ambient environment.  A quick test with a mirror sphere showed that model-viewer uses an environment texture and I was unable to find an option in the documentation to turn on ambient lighting and reflections.  Is there a way to do this, or is this a planned future update for model viewer?

**Comment by elalish** [comment, marked as answer]
Actually this was just added, so it'll be available in our next release.

**Reply by ericwood73** [reply]
Thats great news, thanks.  Will a preview be available that we can start to play with?  Do you have a timeline for the next release?
